packages/frontend-shared/src/locales/en-US.json

```diff
@@ -151,6 +151,15 @@
         "linkText": "average spec durations"
       }
     },
+    "flaky": {
+      "badgeLabel": "Flaky",
+      "severityLow": "Low",
+      "severityMedium": "Medium",
+      "severityHigh": "High",
+      "flakyRate": "{0}% flaky rate",
+      "flakyRuns": "{0} flaky runs / {1} total",
+      "lastFlaky": "Last flaky {0} runs ago"
```

Comment:
Do we need to take singular/plural into account here? Aka is there a case where this could render `1 flaky runs / 1 total` or `Last flaky 1 runs ago`?